finetune/full.py

```diff
@@ -21,24 +23,25 @@
 from lit_gpt.args import EvalArgs, IOArgs, TrainArgs
 from lit_gpt.model import GPT, Block, Config
 from lit_gpt.tokenizer import Tokenizer
+from lit_gpt.data import Alpaca, LitDataModule, apply_prompt_template
 from lit_gpt.utils import (
     CLI,
     check_valid_checkpoint_dir,
     chunked_cross_entropy,
     get_default_supported_precision,
     load_checkpoint,
     num_parameters,
+    CycleIterator,
 )
-from scripts.prepare_alpaca import generate_prompt
 
 
 def setup(
     precision: Optional[str] = None,
     devices: int = 1,
     resume: Union[bool, Path] = False,
+    data: Optional[LitDataModule] = None,
```

Comment:
Found a small bug in jsonargparse: https://github.com/omni-us/jsonargparse/issues/460
So for now, using None as sentinel (maybe that's better anyway)